lib/rubycritic/core/analysed_module.rb

```diff
@@ -19,6 +23,22 @@ def path
       @path ||= pathname.to_s
     end
 
+    def file_location
+      file_path[0...-1].join('/')
```

Comment:
Did we actually ever discuss if we support Windows? If yes, this should be `File::SEPARATOR`.